mindmaps-engine/src/main/java/io/mindmaps/api/ImportController.java

```diff
@@ -151,36 +163,45 @@ private void consumeEntity(Var var) {
         }
     }
 
-    private void consumeRelation(Var var) {
+    private void consumeRelationAndResource(Var var) {
```

Comment:
consumeVarsWithDependencies or something instead? This method is for variables that may depend on previously-loaded variables.